org.opent2t.sample.hub.superpopular/com.insteon.hub/js/common.js

```diff
@@ -0,0 +1,22 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+
+class accessTokenInfo {
```

Comment:
Fixed.

---
In reply to: [82257409](https://github.com/openT2T/translators/pull/107#discussion_r82257409) [](ancestors = 82257409)